docs/docs/running-hello-world.md

```diff
@@ -225,70 +225,19 @@ To generate `host-release.jar`:
     On a Linux machine that meets the [system requirements](enclave-modes.md#system-requirements), you can run your
     app in all the modes the same way as in mock mode.
 
-=== "macOS"
+=== "macOS/Windows"
 
-    You need a Linux environment to run the host in simulation mode. This section describes how to create such an
-    environment using Conclave's Docker integration.
+    Conclave needs a Linux environment to run the host in simulation mode. But you don't need to do anything as long as
```

Comment:
Not sure how clear this is and was before. 
The title is
`macOs/Windows` and the first sentence is "Conclave needs a Linux environment", which sounds odd to me.
Maybe it is not strictly relevant to this PR but I would say something more direct and clear 
```
You need a Linux Docker container to run the host in simulation mode on macOS and Windows.
```

Reply:
Yeah, maybe revert the merging of mac and windows. It's not really relevant to this PR and can be done by John if necessary.